packages/core/content-type-builder/admin/src/components/FormModal/tests/reducer.set-custom-field-data-schema.test.js

```diff
@@ -51,4 +51,58 @@ describe('CTB | components | FormModal | reducer | actions | SET_CUSTOM_FIELD_DA
 
     expect(reducer(initialState, action)).toEqual(expected);
   });
+
+  it("adds a custom field's default options", () => {
```

Comment:
Could we also test that the error gets thrown if we add an invalid root option?

Reply:
Wait sorry, not sure I understand. I think it would be impossible to get here with an invalid root option, right 🤔 . The CustomFields registry would have already thrown an error. That test case is handled in the StrapiApp tests.